React/Views/RCTWrapperViewController.m

```diff
@@ -73,7 +73,25 @@ - (void)viewWillAppear:(BOOL)animated
     }
 
     UINavigationBar *bar = self.navigationController.navigationBar;
-    bar.barTintColor = _navItem.barTintColor;
+    float red, green, blue, alpha;
+    BOOL colorConversionSuccessful = [_navItem.barTintColor getRed:&red green:&green blue:&blue alpha:&alpha];
+    if(colorConversionSuccessful && alpha < 1.0f) {
+      CGRect rect = CGRectMake(0, 0, 1, 1);
+      UIGraphicsBeginImageContextWithOptions(rect.size, NO, 0);
+      [_navItem.barTintColor setFill];
+      UIRectFill(rect);
+      
+      UIImage *backgroundImage = UIGraphicsGetImageFromCurrentImageContext();
+      UIGraphicsEndImageContext();
+      
+      [bar setBackgroundImage:backgroundImage forBarMetrics:UIBarMetricsDefault];
+      bar.shadowImage = [UIImage new];
```

Comment:
We should expose this as a separate prop so that changing the bar background color doesn't automatically toggle the shadow.